pages/en/lb4/Decorators.md

```diff
@@ -0,0 +1,379 @@
+
+  ```ts
+  // my-controller.ts
+  import { authenticate } from '@loopback/authentication';
+  import { inject } from '@loopback/context';
+
+  class MyController {
+    constructor(
+      @inject(BindingKeys.Authentication.CURRENT_USER)
+      private user: UserProfile,
+    ) {}
+
+    @authenticate('BasicStrategy')
+    async whoAmI() : Promise<string> {
+      return this.user.id;
+    }
+  }
+  ```
+
+## Repository Decorators
+
+  As a Domain-driven design concept, 
+  repository means a mediate between your domain object and data mapping layers
+  using a collection-like interface for accessing domain objects. 
+  In framework LoopBack, a domain object is usually a TypeScript/JavaScript Class instance, 
+  and a typical example of data mappting layer module could be a database's node.js driver.
+  
+  LoopBack repository encapsulates your TypeScript/JavaScript Class instance, 
+  and its methods that communicate with your database.
+  It is an interface to implement data persistency. 
+
+  Repository decorators are used for defining models(domain objects) for use with your chosen datasources, 
+  and the navigation strategies among models.
+
+  If you are not familiar with repository related concepts like Model, Entity, Datasource,  
+  please see LoopBack concept [Repositories](#Repositories.htm) to learn more. 
+  
+### Model Decorators
+
+  Model is a Class that LoopBack builds for you to organize the data that 
+  share same configurations and properties.
+  You can use model decorators to define a model and its properties.
+
+#### Model Decorator
+
+  Syntax: `@model(definition?: ModelDefinitionSyntax)`
+
+  Model decorator is a Class decorator.
+  In LoopBack 4, we inherit the model definition format from LoopBack 3,
+  you can find it in [Model definition JSON file](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html).
+  For usage examples, see [Define Models](Repositories.html#define-models)
+
+  *Please note we will elaborate more about model and model definition in #Model.htm,*
+  *and replace the link above with LB4 link*
+
+  By using a model decorator, you can define a model as your repository's metadata,
+  and resolve your repository with Legacy Juggler that defautly comes with a set of CRUD operations bound to your model.
+  A use case can be found in section [Repository decorator](#repository-decorator)
+
+  Or you also have the flexibility to define a model without being resolved with Legacy Juggler.
+  This allows you to implement database access functions by yourself. 
+
+   ```ts
+  // Missing example here
+  // Will be provided in Model.htm
+  // refer to [example code](https://github.com/strongloop/loopback-next-example/blob/master/services/account-without-juggler/repositories/account/models/Account.ts)
+  ```
+
+#### Property Decorator
+
+  Syntax: `@property(definition: PropertyDefinition)`
+
+  It is a Class property decorator, defines the metadata for a Model property.
```

Comment:
Might want to rephrase this to better establish context.
"The property decorator defines metadata for a property on a Model definition." or something similar.